master/internal/cluster/authz_basic_impl.go

```diff
@@ -20,6 +20,13 @@ func (a *MiscAuthZBasic) CanUpdateAgents(
 	return nil, nil
 }
 
+// CanGetAgentInfo returns nil and nil error.
+func (a *MiscAuthZBasic) CanGetAgentInfo(
```

Comment:
we might want this to take in a RP or workspace although we probably won't use it for now. this will could depend on the workspace/rp mapping work. would we want this to behave differently in that world?

Reply:
Good point! Is this for future RP/Workspace work? Do we know if `GetAgents` (`/api/v1/agents`) is RP/Workspace specific?

Reply:
we should talk with the team working on that project. let's chat in slack